### DISCUSSION TITLE
Rootless still in experimental development?

### DISCUSSION THREAD

**Discussion by ncecsc** [k3s-io/k3s#7775] [Q&A] [answered]
Hey all,

I'm looking at running Kubernetes rootless and K3s is one of the options from the official documentation, however K3s is listed as "experimentally supports rootless mode". Is this still the case or is it production-ready?

https://kubernetes.io/docs/tasks/administer-cluster/kubelet-in-userns/#k3s

Thanks in advance!

**Comment by brandond** [comment, marked as answer]
It is still experimental because it has a number of caveats, and we don't see a good path towards production use. See:
* https://docs.k3s.io/advanced#known-issues-with-rootless-mode
* https://github.com/k3s-io/k3s/discussions/7615

It is pretty much stable though, if you are OK with the limitations.

**Reply by ncecsc** [reply]
Thank you, unfortunately running a multi-node cluster is required for my use-case so I'll have to look at other options.
